website/src/content/deployment-examples/commercetools-connect.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Deploying to commercetools Connect
+---
+
+# Prerequisites
+
+Before you get started, you need to have:
+
+- A commercetools Composable Commerce Project ([get started](https://docs.commercetools.com/getting-started)).
+- A Custom Application [configured in the Merchant Center](https://docs.commercetools.com/merchant-center/managing-custom-applications).
+
+# Create your application
+
+1. Install a Connect [starter template](https://docs.commercetools.com/connect/development#install-a-starter-template).
+2. Open the `mc-app` folder in the Connect starter template and [install the Custom Applications starter template](/getting-started#installing-a-starter-template) with the name `sample-custom-application` inside this folder.
```

Comment:
Nit: I guess the name `sample-custom-application` is just a "placeholder" name. The sentence makes it sound like it's the only possible value.

Is there a particular reason to explicitly mention it? Otherwise I would omit this part.

Reply:
I think the original intention might have been to make a tutorial that used this name, but it's mentioned nowhere else in the document so I'll remove it.